App.tsx

```diff
@@ -41,8 +47,20 @@ export default function App() {
     phone_number: "",
     transactions: [],
   };
-  const [user, setUser] = useState(defaultUser);
-  const [userid, setUserId] = useState("X89EpL9f0nNE49PvXp0ERhkO30U2");
+  // const [user, setUser] = useState(defaultUser);
```

Comment:
before we merge can we remove some of these extra comments and console logs just to keep it clean